admin/orders.php

```diff
@@ -267,7 +267,7 @@
         if (file_exists(DIR_FS_CATALOG_MODULES . 'payment/' . $order->info['payment_module_code'] . '.php')) {
           require_once(DIR_FS_CATALOG_MODULES . 'payment/' . $order->info['payment_module_code'] . '.php');
           require_once(DIR_FS_CATALOG_LANGUAGES . $_SESSION['language'] . '/modules/payment/' . $order->info['payment_module_code'] . '.php');
-          $module = new $order->info['payment_module_code'];
+          $module = new $order->info['payment_module_code']();
```

Comment:
Meh. While I won't reject the PR for this, I'm not convinced it's necessary. It's safe to "new up" a class without passing empty parentheses to it. Or, maybe I'm missing some upcoming deprecation that's threatened by the lack of parens?

Reply:
Comes from here I think:
https://www.php-fig.org/psr/psr-12
4-classes-properties-and-methods

Reply:
Ya, I suppose that's probably what the IDE is holding to.